lib/msal-core/src/utils/Constants.ts

```diff
@@ -52,17 +50,9 @@ export class Constants {
 
     static get interactionTypeRedirect(): InteractionType { return "redirectInteraction"; }
     static get interactionTypePopup(): InteractionType { return "popupInteraction"; }
+    static get IN_PROGRESS(): string { return "InProgress"; }
```

Comment:
IN_PROGRESS -> inProgress(). Consistency in code style :)

Reply:
I thought about it, but left it alone as I did not want to change it in UAA. Guess it makes sense. Changing it.

Reply:
Done.